__tests__/components/Modal.test.jsx

```diff
@@ -39,4 +39,8 @@ describe('Modal tests', () => {
   it('should render modal content', () => {
     expect(wrapper.find('ModalContent')).toHaveLength(1);
   });
+
```

Comment:
I'm curious, is this testing a styled component for 'overflow' or is it testing 'css' (which I don't even know if jest can do)?